app/lib/methods/audioFile.ts

```diff
@@ -0,0 +1,65 @@
+import * as FileSystem from 'expo-file-system';
+import { PermissionsAndroid } from 'react-native';
+
+import i18n from '../../i18n';
+import { store } from '../store/auxStore';
+import { isAndroid } from './helpers';
+import log from './helpers/log';
+
+// It should be necessary, but in my tests, even uninstalling it didn't ask for permission.
+// Maybe it could be the android version?
+// Call when entering room view.
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const askForPermissionToSaveFiles = async () => {
```

Comment:
I think you don't have to, because `documentDirectory` is controlled by the app only.
You would need a permission like that to save on `Downloads` folder, for example.